libs/sis-service/src/test/java/uk/gov/di/ipv/core/library/sis/pact/ContractTest.java

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
Sorry to be a pain, could we pull this commit out to a separate BAU PR (given it's not related to this ticket)